.pre-commit-config.yaml

```diff
@@ -1,11 +1,22 @@
 ---
-
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v2.2.3  # Use the ref you want to point at
-  hooks:
-  - id: trailing-whitespace
-  - id: check-yaml
-  - id: end-of-file-fixer
-  - id: flake8
-  - id: trailing-whitespace
-  - id: check-executables-have-shebangs
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0  # Use the ref you want to point at
+    hooks:
+    - id: trailing-whitespace
+    - id: check-yaml
+    - id: end-of-file-fixer
+    - id: flake8
+    - id: trailing-whitespace
+    - id: check-executables-have-shebangs
+  - repo: https://github.com/python/black
+    rev: 19.3b0
+    hooks:
+      - id: black
+        language_version: python3
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.8
+    hooks:
+      - id: flake8
+        additional_dependencies:
+          - flake8-black>=v0.1.0
```

Comment:
I don't think you need this here, running black + flake8-black I believe is redundant

Reply:
Agreed. I don't even really think `flake8-black` is a thing that makes sense but :shrug: